MoneyAPP/moneyapp/lib/source/api_source.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:http/http.dart' as http;
+import 'dart:convert';
+import 'package:logger/logger.dart';
+
+final Logger logger = Logger();
+Future<int> getRandomNumber() async {
+  final url = Uri.parse(
```

Comment:
- [ ] do it with retrofit, like this
```dart
import 'dart:convert';

import 'package:dio/dio.dart';
import 'package:injectable/injectable.dart';
import 'package:retrofit/retrofit.dart';

part 'api_data_source.g.dart';

@RestApi(parser: Parser.MapSerializable)
@lazySingleton
abstract class ApiDataSource {
  @factoryMethod
  factory MuxDataSource() {
    return _MuxDataSource(Dio(), baseUrl: 'https://api.allorigins.win/get?url=http://www.randomnumberapi.com/api/v1.0');
  }

  // google how to set cacheBust
  @GET('/random')
  Future<void> getRandomNumber(@Path() String assetId);
}
```
- [ ] implement logger as Dio interceptor

Reply:
Fixed.